python/tvm/script/builder/tir/var.py

```diff
@@ -142,4 +142,4 @@ def __getitem__(self, keys) -> Buffer_:
         return self(*keys)  # pylint: disable=no-member # type: ignore
 
 
-Buffer = BufferProxy()
+Buffer = buffer_decl = BufferProxy()
```

Comment:
This may not be right, because if so, users may use `buffer_decl[]` to create buffers, which looks weird